python/src/py/keyvi/__init__.py

```diff
@@ -17,6 +17,8 @@
 limitations under the License.
 '''
 
+__version__ = '0.3.1dev'
```

Comment:
I think Pep440 want to have a number after dev, e.g. '0.3.1dev0'

https://www.python.org/dev/peps/pep-0440/

Reply:
as said what about:
```
__version_major__ = 0
__version_minor__ = 3
__version_patch__ =  1
__version_dev__ = 0

# to be set externally
__version_dev_mode__ = true 

if __version_dev_mode__: 
    __version__ = "{}.{}.{}dev{}".format(__version_major__, __version_minor__, __version_patch__, __version_dev__)
else:
   __version__ = "{}.{}.{}".format(__version_major__, __version_minor__, __version_patch__)
```

`__version_dev_mode__ ` should then be false for releases. Some little logic would be required to set this. Maybe `setup.py` can write or touch a file for that.